### PR TITLE
Updating the list of alums, and my personal page

### PR DESCRIPTION
**Pull request by brancokm**
Creating a pull request to update both my personal page and to update the alum section of the website.

### CODE REVIEW

**Review by massimolarsen — APPROVED** [review]
Looks good to me

File: users/brancokm/index.md [hunk 1]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
-I have previously worked as an intern at Lawrence Livermore National Laboratory, and am always interested in hearing new project ideas. 
-Please contact me at: Brancokm@oregonstate.edu 
\ No newline at end of file
+Email me: [brancokm@oregonstate.edu](mailto:brancokm@oregonstate.edu)
+
+Add me on [LinkedIn](https://www.linkedin.com/in/michael-b-katcher/)
+
+I am a second year graduate student at Oregon State University under Dr. Todd Palmer. I am currently funded by a DOE-NE, NEUP fellowship. My research will center on how best to design reactors and criticality expeirments for specific purposes, looking to appropriately account for qualitative and quantative factors in the design of these systems. My area of interest is in Gen IV reactors, and my work will cater to those platforms specifically. Previous experience has included criticality safty, sensitivty/uncertainty quantification in nuclear data with respect to k-eff, and system design of integral benchmarks experiments.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Sensitivity and quantitative are misspelled.

File: users/brancokm/index.md [hunk 2]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
-Please contact me at: Brancokm@oregonstate.edu 
\ No newline at end of file
+Email me: [brancokm@oregonstate.edu](mailto:brancokm@oregonstate.edu)
+
+Add me on [LinkedIn](https://www.linkedin.com/in/michael-b-katcher/)
+
+I am a second year graduate student at Oregon State University under Dr. Todd Palmer. I am currently funded by a DOE-NE, NEUP fellowship. My research will center on how best to design reactors and criticality expeirments for specific purposes, looking to appropriately account for qualitative and quantative factors in the design of these systems. My area of interest is in Gen IV reactors, and my work will cater to those platforms specifically. Previous experience has included criticality safty, sensitivty/uncertainty quantification in nuclear data with respect to k-eff, and system design of integral benchmarks experiments. 
+
+I have experience in both determinstic and Monte-Carlo neutron transport methods, but tend to favor Monte-Carlo methods and have used tools from Serpent and MCNP, to open source equivalents like OpenMC. My experience in determinstic methods have all been within the MOOSE framework of tools.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Deterministic is misspelled.

File: users/brancokm/index.md [hunk 3]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
+
+I completed my undergraduate studeis at Oregon State in June of 2021. During undergrad I was involved with determinstic simulations of Molten Salt Reactors, modeling of gamma-detectors, simulation of fuel in a Pebble Bed Reactor and a study regarding how Monte-Carlo methods may be used to predict transmutations in the structural material of a LWR reactor and how potential transmuations might influence material properties. Minored in Material Science.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Studies and transmutations are misspelled.

File: users/brancokm/index.md [hunk 4]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
+
+# Work Experience
+## Sensitivty and Uncertainty Quantification, Terrapower
+- Calculated the sensitivty in k-eff of multiple advanced reactors platforms to nuclear data uncertainty, and calculated total nuclear data uncertainty in k-eff for the TerraPower reactor platforms. Calculations performed in MCNP for multiple data sets.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Sensitivity is misspelled.

File: users/brancokm/index.md [hunk 5]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
+- Calculated the sensitivities and uncertainties in k-eff for a suite of benchmarks potentially relevant to the TerraPower platforms. Calculations performed in MCNP for multiple data sets.
+- Performed an evaluation how related existing benchmarks with similar uncertainties were to the proposed TerraPower reactors, calculating a representativity factor between the platforms of inerest and each benchmark.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Interest is misspelled.

File: users/brancokm/index.md [hunk 6]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
+
+## Criticality Safety, Lawrence Livermore National Laboratory
+- Participated in the design of an integral experiment as part of the LLNL Thermal-Epithermal eXperimental (TEX) campaign, using HEU plates and moderators of various materials surrounded by a polyethelene reflector. Designed an experiment to cater to a customers specific material needs. Performed criticality calculations, selected optimized designs, showed sensitivity and uncertainty of the system with respect to both manufacturing tolerances and systematic uncertainty.
+- Carried out a study on correlation and covariance calculations in legacy criticality experiments to propose how covariance and correlation coefficients can potentially. Quantified the correlations of experiments with known relations that had yet to be explored. Culiminated in a paper that was accepted for presentation and publication for the Nuclear Criticality Safety topical meeting at the ANS National conference in 2022.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Culminated is misspelled.

File: users/brancokm/index.md [hunk 7]
```diff
@@ -6,6 +6,29 @@ author: Michael Branco-Katcher
...
+- Work performed through a combination of Python scripts and MCNP.
+
+## National Security Engineering, Lawrence Livermore National Laboratory
+- Utilized SCALE 6.2.3. to calculate spent fuel isotopics from reactors in various configurations, at different decay times.
+- Performed an analysis on the operations of a reactor to evaluate non-proliferation instrumation.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: Instrumentation is misspelled.

**Review by aaronjamesreynolds — COMMENTED** [review]
Got a couple corrections here. @brancokm might be worth copy and pasting your index.md text into a word processor to check for any misspellings I missed.

**Comment by brancokm** [comment]
I have fixed the errors Aaron mentioned. Should be good for merging.

**Review by clemekay — APPROVED** [review]
There's a missing apostrophe but it doesn't matter to anyone but me

**Pull request merged**